src/components/Footer.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+    <div className="footer p-8 w-full max-w-2xl mx-auto">
+      <p className="disclaimer pb-6 text-sm">
+        This website does not provide medical advice. The information, including but not limited to,
+        text, graphics, images and other materials contained on this website are for informational
+        purposes only. No material on this site is intended to be a substitution for professional
+        veterinarian advice, diagnosis or treatment. Always seek the advice of your veterinarian
+        with any questions you may have regarding how to feed or medicate your pet. Never disregard
+        professional advice or delay in seeking it because of something you have read on this
+        website!
+      </p>
+      <p className="text-sm">
+        Designed and developed by{' '}
+        <a href="https://github.com/Trinityyi" className="underline">
+          Isabelle Viktoria Maciohsek
+        </a>{' '}
+        &copy; {new Date().getFullYear()} /{' '}
+        <a href="https://github.com/Trinityyi/can-my-dog-eat" className="underline">
+          Source code
+        </a>
+        <br /> Built with{' '}
+        <a href="https://reactjs.org/" rel="nofollow noreferrer">
```

Comment:
[minor] I suppose you could add a `title` attribute here for consistency with the rest of the links.